cardano-node.cabal

```diff
@@ -11,43 +11,70 @@ cabal-version:  >= 1.10
 extra-source-files: README.md, ChangeLog.md
 
 library
+  hs-source-dirs:      src
+
   exposed-modules:
-      Lib
+                       Lib
```

Comment:
Surely we can namespace this now rather than later? Like `Cardano.Node` or something?